### ISSUE TITLE
v2.0.4 can't append symbol when runtime

### ISSUE THREAD

**Issue by pfdgithub** [kisenka/svg-sprite-loader#101]
svg-sprite-loader v0.3.1
```
Sprite.prototype.appendSymbol = function (content) {
  var symbol = this.wrapSVG(content, Sprite.symbolTemplate()).childNodes[0];

  this.svg.querySelector('defs').appendChild(symbol); // append symbol when runtime
  if (this.browser.name === 'firefox') {
    FirefoxSymbolBugWorkaround(this.svg);
  }
};
```
svg-sprite-loader v2.0.4
```
Sprite.prototype.add = function add (symbol) {
  var ref = this;
    var symbols = ref.symbols;
  var existing = this.find(symbol.id);

  if (existing) {
    symbols[symbols.indexOf(existing)] = symbol;
    return symbol;
  }

  symbols.push(symbol);

  // fix start
  if(ref.isMounted){
    // ref.node.appendChild(parse(symbol.content)); // it's not work. why?
    ref.node.insertAdjacentHTML('beforeend', symbol.content); // it's work
  }
  // fix end

  return symbol;
};
```

**Comment by kisenka** [comment]
@pfdgithub which browser is used?

**Comment by pfdgithub** [comment]
@kisenka chrome 55

**Comment by kisenka** [comment]
Fixed in `svg-sprite-loader@2.1.0`
